cpp/oneapi/dal/algo/kmeans/test/ccl_spmd.cpp

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+* Copyright 2022 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/spmd/ccl/communicator.hpp"
+
+#define PARALLE_BACKEND dal::preview::spmd::backend::ccl
```

Comment:
It seems there is a typo: 'PARALLE_BACKEND'. Should be 'PARALLE**L**_BACKEND'

Reply:
Fixed